pubsub/tests/unit/pubsub_v1/publisher/batch/test_thread.py

```diff
@@ -72,7 +72,7 @@ def test_init_infinite_latency():
     assert batch._thread is None
 
 
-@mock.patch.object(threading, 'Lock')
+@mock.patch.object(threading, 'RLock')
 def test_make_lock(Lock):
```

Comment:
If we *are* going to use an `RLock` (see above), then change the name of the argument to match.